index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta
+  name="description"
+  content="This is a website about Stanmega's Web Development Courses."
+/>
+<title>Stanmega-Tech Web Development Courses</title>
+<link rel="stylesheet" href="./style.css" type="text/css" />
+<script
+  src="https://kit.fontawesome.com/e1fe8e6842.js"
+  crossorigin="anonymous"
+></script>
+<script src="menu-mobile.js" defer></script>
+<script src="featured-profs.js" defer></script>
+</head>
+<body>
+  <!--Navigation Bar Design-->
+  <nav>
+    <div class="topmost-bar">
+      <span><i class="fas fa-times fa-2x" id="exit-button"></i></span>
+      <ul>
+         <li><i class="fa fa-facebook"></i></li>
+         <li><i class="fab fa-twitter"></i></li>
+         <li><a href="#">English</a></li>
+         <li><a href="#">My Page</a></li>
+         <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="top-navbar">
+      <div class="logo">
+        <i class="fa fa-university fa-2x" aria-hidden="true">
+        <a href="./index.html"></a></i>
+        <h2>Stanmega-tech <span>Web Development</span></h2>
+      </div>
+
+      <ul class="nav-icons">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+      </ul> 
+      <div id="campaign">
+      <i class="fa fa-university" aria-hidden="true" >
+      <a href="#">SMT Campaign</a></i>
+    </div>
+  </div>
+  </nav>
+  <!--Header design-->
+  <header>
+    <span class="menu">
+      <i class="fas fa-bars fa-2x" id="menu"></i></span>
+    <div class="headline">
+    <span id="welcome">"Hello! Welcome to Stanmega-Tech"</span>
+    <h1 class="h1text">
+      WEB DEVELOPMENT
+      <span class="h1-span">COURSE</span>
+      <span class="h1-span">2022</span>
+    </h1>
+    <p>
+      Do you want to become a web developer? If yes, then this course is
+      just for you. The program's duration is 10 months of intensive learning with practicals. We start a new class every 2 months, so if you don't
+      want to miss the start date for the next class, then sign up and join
+      us now.
+    </p>
+    <h2 id="next-class"
+      >Our next class starts:
+      <span id="start-date">2022.05.23.(MON)</span></h2>
+    <span id="location"><a href="#"> Stanmega-Tech online Link</a></span>
+    <button type="button" class="access">Get Access Now</button>
+  </div>
+</header>
+
+<!--Our Curriculum-->
+<main>
+<section id="program">
+  <h2 class="text">Our Programs | Curriculum</h2>
+  <div class="text-line"></div>
+  <div id="languages-container">
+    <div class="technologies">
+       <i class="fab fa html5 fa-3x"></i>
```

Comment:
- [ ] The HTML icon is not being shown right now, maybe there is a typo within the code about it, so please make sure to check and work on this issue.